packages/thirdweb/src/react/core/hooks/connection/ConnectButtonProps.ts

```diff
@@ -255,6 +255,19 @@ export type ConnectButton_detailsModalOptions = {
    */
   hideDisconnect?: boolean;
 
+  /**
+   * Hide the wallet image on the "Switch Wallet" button in the `ConnectButton` Details Modal.
+   *
+   * By default it is `false`
+   * @example
+   * ```tsx
+   * <ConnectButton detailsModal={{
+   *  hideWalletImage: true
+   * }} />
+   * ```
+   */
+  hideWalletImage?: boolean;
```

Comment:
do you actually need this? worried its gonna look the layout look very weird

Reply:
Nice to have - it just covers part of the avatar with its current positioning. I'm not sure if there's some other use of this that I'm missing that would mess up the layout.

<img width="242" alt="Screenshot 2024-08-26 at 4 04 15 PM" src="https://github.com/user-attachments/assets/84661524-bc9f-47bf-95f9-651589a25afa">

Reply:
i'll combine this with `hideSwitchWallet` instead, if you turn off the button behavior, then it hides the icon too